### ISSUE TITLE
Add option for separating out technical replicates

### ISSUE THREAD

**Issue by nboley** [kundajelab/bds_atac#17]
- PCR duplicate should happen on each technical replicate
- The the bams should be merged with a RG assignment atatched
   - we should allow the user to specify custom read group meta data

**Comment by chrisprobert** [comment]
@nboley: Just to clarify: a technical replicate here is a unique sample/lane combination? So a library sequenced on multiple lanes has a technical replicate for each sample in each lane?

**Comment by akundaje** [comment]
I think it's dangerous to do excessive inference from read names etc.
Platforms or conventions often change and then all of this breaks down.
Best to have users be explicit about what tech reps, lanes, bioreps and
adapter sequences.

**Comment by chrisprobert** [comment]
@akundaje Yeah, I think some type of sample sheet to specify parameters for all replicates would be a good solution, perhaps combined with automated sanity checking (e.g. if the user-specified adapter sequence is clearly wrong).

**Comment by akundaje** [comment]
Agreed
